.gitignore

```diff
@@ -40,6 +40,10 @@ Temporary Items
 ######
 .idea/
 
+# TEMP
+######
+temp/
+
```

Comment:
Can we remove this - this is better left to your global .gitignore maybe?

Reply:
I will, 
I'll delete the temp folder when I am done with the files in there.
I needed it to stick around for a bit so that I could take a look at the generated USD files in maya/houdini.